src/state/slices/tableSlice.tsx

```diff
@@ -38,7 +43,7 @@ export const tableSlice = createSlice({
     // Use the PayloadAction type to declare the contents of `action.payload`
     selectColumn: (state, action: PayloadAction<string>) => {
       // if it's the timestamp column, add to the beginning of the array
-      if (action.payload === 'timestamp') {
+      if (action.payload === timeChannelName) {
         state.selectedColumnIds.unshift(action.payload);
```

Comment:
Is it me or is unshift a terrible name for what it does?

Reply:
It is, hence why I added the comment to make it clear that that's what this is doing 😅

Reply:
If it wasn't for the comment I would have no idea what it does without googling.